app/controllers/api/v1x0/graphql_controller.rb

```diff
@@ -17,7 +17,7 @@ def overlay
         {
           "^.*$" => {
             "base_query" => lambda do |model_class, _ctx|
-              policy_scope(model_class)
+              policy_scope(model_class.all)
```

Comment:
@hsong-rh What prompted this change? policy_scope takes in a relationship and not a class?

Reply:
class should be sufficient and preferred.

Reply:
Both relation and class are ok for policy_scope. The change here just keeps in sync with other places.

Reply:
Changed to use class